lib/response.js

```diff
@@ -51,7 +51,9 @@ function responseError (res, code, detail, msg) {
     code: code,
     detail: detail,
     msg: msg,
-    useCDN: config.usecdn
+    useCDN: config.usecdn,
+    allowAnonymousEdits: false,
+    signin: false
```

Comment:
Shouldn't these newly added options get a useful value instead of being statically set to false?

Reply:
in their context we only have a response object, `res`. If that object can allow them to be derived from something -- fine. I was not able to find out what I can derive them from.

As this is just about displaying an error message, I think it is OK to not display the edit/publish buttons but we have to have variables defined to avoid throwing an error in the EJS code that references them.